app/lib/submission_builder/ty2022/states/ny/documents/it201_additional_dependents.rb

```diff
@@ -0,0 +1,41 @@
+module SubmissionBuilder
```

Comment:
i think this might be in the wrong place...? this folder, i *think*, is for xml builders but this is a pdf builder and should go in `app/lib/pdf_filler/`

Reply:
also, a unit spec for this would probably be good